### DISCUSSION TITLE
Is it possible to expand relations in the Admin UI?

### DISCUSSION THREAD

**Discussion by trey** [pocketbase/pocketbase#1357] [Q&A] [answered]
I'm trying to create a list of books and associated authors, and I'd like to be able to see the related record in the list in the admin instead of the record IDs.

<img width="637" alt="Two rows representing Stephen King books. The authors and series entries are random strings representing database IDs" src="https://user-images.githubusercontent.com/435/209375057-63920a13-71d6-450f-b2c3-548b92307b71.png">

In this example, I'd love to be able to see `Stephen King` in the authors field instead of `0qmft84qi11vu3p`. I realize this is possible programmatically, but it would be awesome to see it in the admin as well for easier reference.

**Comment by ganigeorgiev** [comment, marked as answer]
No it is not possible at the moment, but this will be fixed in v0.12/v0.13 with https://github.com/pocketbase/pocketbase/issues/976 and its related issues (_if you haven't already, you can check the currently planned issues priority in https://github.com/orgs/pocketbase/projects/2_).

**Reply by trey** [reply]
Thank you!
